### PR TITLE
Update "deprecate" to import from @ember/debug

### PR DESCRIPTION
**Pull request by snewcomer**
ref https://github.com/simplabs/ember-simple-auth/pull/2315#issuecomment-904220426

### CODE REVIEW

File: packages/ember-simple-auth/package.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
     "ember-sinon": "~5.0.0",
     "ember-source": "~3.18.0",
     "ember-source-channel-url": "^2.0.1",
-    "ember-test-selectors": "^4.0.0",
+    "ember-test-selectors": "^6.0.0",
```
Comment (marcoow) [inline packages/ember-simple-auth/package.json:65]: this seems unrelated?

Comment (snewcomer) [reply packages/ember-simple-auth/package.json:65]: I was working on getting the tests passing.  Looks like they are failing on the main branch so hopefully I can resolve here 👉 

https://github.com/simplabs/ember-simple-auth/pull/2319

**Review by BobrImperator — APPROVED** [review]

**Review by marcoow — COMMENTED** [review]
Would it make sense to update [all of the usages](https://github.com/simplabs/ember-simple-auth/search?q=%40ember%2Fapplication%2Fdeprecations) of the old import in this PR?

**Comment by snewcomer** [comment]
oh my how I missed that :). Fixed

**Comment by marcoow** [comment]
@snewcomer there are some linter failures now because duplicated imports…

**Review by marcoow — APPROVED** [review]
